### ISSUE TITLE
Switch to read-only datasource

### ISSUE THREAD

**Issue by duxbuse** [duxbuse/ninthage-data-analytics#155]
After the creation of the new data set that is only views and hence only readonly to ensure that now woopsies occur It would be good to migrate tableau to use this instead of the table directly

Note @SanderND You may have already done this if so please close

**Comment by SanderND** [comment]
The Tableau dashboard has been switched already. I'm using the old one to build the new SQL code so please keep it alive for a while. Once the new code is finished i copy the code and delete the old interface view to avoid confusion.
